ietf-layer0-types-ext.yang

```diff
@@ -219,6 +219,16 @@ module ietf-layer0-types-ext {
       "Amplifiers and Transceivers Power in dBm.";
   }
 
+  typedef snr {
+    type decimal64 {
+        fraction-digits 3;
+      }
+    units "dB";
+    description
+      "Signal to noise ratio in 0.1 nm";
+  }

```

Comment:
Please check and propose an alignment with the previous definition which is used by https://tools.ietf.org/html/draft-ietf-ccamp-optical-impairment-topology-yang-05 but was committed to the transponder-model-types branch and not to the master branch and that we have just merged with PR#13

```
  typedef snr {
    type decimal64 {
      fraction-digits 2;
    }
    units "dB@0.1nm";
    description
      "(Optical) Signal to Noise Ratio measured over 0.1 nm
       resolution bandwidth";
  }

```

Co-authored-by: sergiobelotti sergio.belotti@nokia.com

Reply:
Addressed with https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/11/commits/9e2c4758a623ed7879d84d055d58807d3df2bc51

Reply:
Addressed with https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/11/commits/9e2c4758a623ed7879d84d055d58807d3df2bc51